components/software_sensor_iso_supported.mdx

```diff
@@ -0,0 +1,79 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+This sensor is included in the Clearpath Robotics operating system that is installed on all our robots.
```

Comment:
Maybe this is a minor nitpick, but I wouldn't really call our ISO an "operating system". Other possible options to consider:

- Clearpath Robotics ISO
- Clearpath Robotics Bundle
- Clearpath Robotics Installation Image
- Clearpath Robotics Installation Bundle
- Clearpath Robotics System Image

Reply:
My vote is `Clearpath Robotics System Image`

Reply:
I changed this to 'Clearpath Robotics System Image'.